src/components/work/CancelBookingModal.tsx/CancelBookingModal.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { ModalHeader, Modal, ModalBody, ModalFooter, Button } from 'reactstrap';
+import { GUEST_CANCEL_BOOKING, Booking, Currency, GET_GUEST_SORTED_BOOKINGS } from 'networking/bookings';
+import { graphql, compose } from 'react-apollo';
+import { differenceInDays } from 'date-fns';
+import { cancel, loadWeb3 } from 'utils/web3';
+import { AlertProperties } from 'components/work/Alert/Alert';
+import LoadingPortal from 'components/work/LoadingPortal';
+import { getFriendlyErrorMessage } from 'utils/validators';
+
+interface Props {
+  booking: Booking;
+  cancelBooking: (booking: Booking) => Promise<Booking>;
+  handleModalAction: () => void;
```

Comment:
For consistency with React naming conventions, I'd expect this to be `onModalAction` at the interface level and `handleModalAction` at the implementation level, for usage like:

    <CancelBookingModal onModalAction={handleModalAction} etc... />

Reply:
that was bugging me in the back of my head. I'll try to remember or refer back to this in the future. tyty!